website/events/templates/events/index.html

```diff
@@ -16,26 +16,15 @@
 
 {% block page_title %}
     {% trans "Calendar" %}
-    <a href="{% url 'events:ical-'|add:LANGUAGE_CODE %}"
+    <a href="{% url 'events:ical-help' %}"
        target="_blank"
-       class="btn btn-primary first-right"
+       class="btn btn-primary first-right d-flex align-items-center"
        data-bs-toggle="tooltip"
        data-bs-placement="top"
-       title="{% trans 'iCal feed (all events)' %}"
+       title="Configure iCal feed"
     >
-        <i class="fas fa-calendar-alt"></i>
+        <i class="fas fa-calendar-alt me-2"></i>iCal feed
```

Comment:
"iCal feed" is quite deep into the realm of jargon. Calling the button something like "import calendar" or "synchronize calendar" would probably be more user-friendly.

Even though Thalia's audience is CS people, we should not assume familiarity with this kind of jargon; consider 2FA setup for an example where this led to issues.